README.md

```diff
@@ -163,13 +164,13 @@ This SDK provides a basic serialization for request/response objects but also fo
     ```
 
 
-### Debug Java application or Dapr's Java SDK
+### Debug a Java application or Dapr's Java SDK
 
-**In IntelliJ Community Edition, consider [debugging in IntelliJ](https://docs.dapr.io/developing-applications/ides/intellij/).**
+**In IntelliJ Community Edition, consider [debugging in IntelliJ](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html).**
 
-**In Visual Studio Code, consider [debugging in Visual Studio Code](https://docs.dapr.io/developing-applications/ides/vscode-debugging/).**
+**In Visual Studio Code, consider [debugging in Visual Studio Code](https://code.visualstudio.com/docs/editor/debugging).**
```

Comment:
Is there a reason to change this link?

Reply:
that original link is a 404: Not Found

Reply:
https://docs.dapr.io/developing-applications/local-development/ides/vscode/